stdlib/public/SDK/Intents/INSearchCallHistoryIntent.swift

```diff
@@ -14,29 +14,25 @@
 import Foundation
 
 #if os(iOS) || os(watchOS)
-@available(iOS 10.0, watchOS 3.2, *)
+@available(iOS 11.0, watchOS 4.0, *)
 extension INSearchCallHistoryIntent {
+  @available(iOS 11.0, watchOS 4.0, *)
```

Comment:
This is a behavior change, even if it wasn't working before. Do we really not want to support this entry point on pre-iOS-11 systems?